### PR TITLE
Voron V0 Mean Well Power Supply Bracket & SSR Mount

### PR DESCRIPTION
**Pull request by DanJunior78**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.

#### Which mods/configurations/profiles are added by this PR?

Two brackets to prevent using double side tape. 

- Mean Well power supply bracket
- SSR Mounting bracket

including the BOM

#### How was it tested? 

see pictures in the folders. It´s assembled on my V0.1

#### Any background context you want to provide?
---
#### Screenshots (if appropriate)
in the readme accessible

#### Further notes
---

### CODE REVIEW

File: printer_mods/DaGlo/Voron_V0_MW_PS_Bracket/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Voron V0 Mean Well power supply bracket (Funssor Kit)
+
+This bracket works for the panel kit from Funssor from AliExpress, which has already some holes on the bottom bracket.
+
+"Funssor upgrade Voron 0 V 0,1 acryl pannel kit für Voron 0,1 3D drucker DIY kit"
+https://de.aliexpress.com/item/1005002596904062.html?spm=a2g0s.9042311.0.0.27424c4djYAPzu
```
Comment (KiloQubit) [inline printer_mods/DaGlo/Voron_V0_MW_PS_Bracket/README.md:6]: Please remove the session information from the link (everything after `.html`), it should be just `https://de.aliexpress.com/item/1005002596904062.html`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

The bottom of these holes in `Voron_V0_SSR_Mount.stl` have unprintable overhangs, you can fix them for example by adding a 45 degree chamfer:

Screen Shot 2021-06-19 at 19 50 11

**Comment by DanJunior78** [comment]
Hello, i´ve fixed it. Did the 45° cut to the outter wall. I´ve printed it with my HevoRS. Little overhangs are not a big deal for it on ASA/PETG. But, yes, thanks for your review. I really appreciate the reviews of you guys, doing a really, really good job on it. :) (i always miss some little details, ah...sure...it´s a V0...the nicest little detail which is able to print, lol)

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**